frontend/ui/widget/imagewidget.lua

```diff
@@ -81,12 +81,16 @@ local ImageWidget = Widget:new{
     -- If scale_for_dpi is true image will be rescaled according to screen dpi
     scale_for_dpi = false,
 
-    -- When scale_factor is not nil, native image is scaled by this factor
-    -- (if scale_factor == 1, native image size is kept)
-    -- Special case : scale_factor == 0 : image will be scaled to best fit provided
-    -- width and height, keeping aspect ratio (scale_factor will be updated
-    -- from 0 to the factor used at _render() time)
+    -- When scale_factor is not nil, native image is scaled by this factor,
+    --   (if scale_factor == 1, native image size is kept)
+    --   Special case: scale_factor == 0 : image will be scaled to best fit provided
+    --   width and height, keeping aspect ratio (scale_factor will be updated
+    --   from 0 to the factor used at _render() time)
+    -- If scale_factor is nil and the aspect ratios of the image and the widget don't differ
+    --   more than stretch_limit_percentage, the image will be stretched to fill widget.
```

Comment:
Hmm, I think I wrote this right. As said if the ratios differ less than limit_percentage (this means implicitely that limit percentages has to be a number ) then we get the image streched.
 I will note that explicitely.

Reply:
It's just the wording:
With `If scale_factor is nil and the aspect ratios of the image and the widget don't differ more than stretch_limit_percentage`, one would wonder (at least, I did :)  what happens if stretch_limit_percentage is not provided.
`If scale_factor is nil and stretch_limit_percentage is provided, ...` is just enough to avoid any doubts.